_posts/en/newsletters/2020-05-27-newsletter.md

```diff
@@ -0,0 +1,178 @@
+version 0.20.*
+
+- [Bitcoin Core #18956][] uses the API on Windows systems to require
+  Windows 7 or later.  All release notes since the October 2018 release
+  of [Bitcoin Core 0.17][0.17 compat] have announced that the only
+  supported versions of Windows are version 7 or later.
+
+- [Bitcoin Core #18861][] prevents the node from replying to a P2P
+  protocol `getdata` request for a transaction that it hasn't yet
+  announced to the requesting peer.  The prevents surveillance nodes from
+  circumventing Bitcoin Core's existing privacy-enhancing behavior of
+  waiting a slightly different amount of time for each peer (or group of
+  peers) before announcing new transactions to them, causing each
+  transaction to propagate across the network using a different path.
+  Randomizing the propagation path of each transaction makes it less
+  reliable for surveillance nodes to assume that the first node announcing
+  a transaction was the first node to receive it.
+
+- [Bitcoin Core #17681][] allows the wallet to internally derive new
+  addresses for a [BIP32][] HD wallet seed even after that seed is no
+  longer the wallet's active seed.  This makes it safe to switch to a
+  new HD seed with the `sethdseed` (set HD seed) RPC even while the node
+  is performing an initial block chain download, such as when restoring
+  a wallet backup on a newly-started node---the updated code ensures the
+  wallet will see any payments to addresses previously derived from the
+  old HD seed.
+
+- [Bitcoin Core #18895][] updates the RPCs that return data about
+  individual transactions in the mempool (e.g. `getrawmempool` and
+  `getmempoolentry`) with an `unbroadcast` field that indicates whether
+  or not any of the local node's peers have requested a copy of the
+  transaction (see [Newsletter #96][news96 unbroadcast] for a summary of
+  broadcast tracking).  Additionally, the `getmempoolinfo` RPC is
+  updated with an `unbroadcastcount` field indicating the number of
+  unbroadcast transactions.  For privacy, the broadcast status of a
+  transaction is only tracked if it was submitted by either the node's
+  wallet or the `sendrawtransaction` RPC.
+
+- [Bitcoin Core #18677][] adds a new `--enable-multiprocess` build
+  configuration option that will produce additional binaries alongside
+  the existing `bitcoind` and `bitcoin-qt` binaries.  For now, the only
+  difference between the new and old binaries is their name.  However,
+  if [PR #10102][Bitcoin Core #10102] is merged, the new binaries will
+  split the functions of node, wallet, and GUI into separate executables
+  that communicate with each other when necessary.  The build option is
+  currently disabled by default.  See also [Newsletter #39][news39
+  multiprocess] for the last time we wrote about the multiprocess
+  sub-project.
+
+- [Bitcoin Core #18594][] allows the `bitcoin-cli` command to print the
+  balances of each wallet loaded in multiwallet mode using the
```

Comment:
Perhaps: s/balances/balance/ since each wallet has one balance.

Note that this is the first time that a RPC/CLI command can return multiple wallet balances. 

It was initially proposed as a server-side RPC implementation, but more discussion is needed on the interface to adopt for a general multiwallet API.